core/src/test/java/org/infinispan/stream/BaseStreamTest.java

```diff
@@ -1489,6 +1489,48 @@ public void testDoubleAllMatch() {
       assertFalse(createStream(entrySet).mapToDouble(toDouble).allMatch(i -> Math.floor(i) == i));
    }
 
+   public void testSkip() {
+      Cache<Double, String> cache = getCache(0);
+      int range = 10;
+      // First populate the cache with a bunch of values
+      IntStream.range(0,range).mapToDouble(value -> value).forEach(i -> cache.put(i, i + "-value"));
+      CacheSet<Map.Entry<Double, String>> entrySet = cache.entrySet();
+
+      int cnt = range;
+      for(int i = 0;i<cnt;i++){
```

Comment:
Ah yes my IntelliJ settings are probably off.

On Fri, Apr 29, 2016, 16:28 wburns <notifications@github.com> wrote:

> In core/src/test/java/org/infinispan/stream/BaseStreamTest.java
> <https://github.com/infinispan/infinispan/pull/4288#discussion_r61585868>:
>
> > @@ -1489,6 +1489,48 @@ public void testDoubleAllMatch() {
> >        assertFalse(createStream(entrySet).mapToDouble(toDouble).allMatch(i -> Math.floor(i) == i));
> >     }
> >
> > +   public void testSkip() {
> > +      Cache<Double, String> cache = getCache(0);
> > +      int range = 10;
> > +      // First populate the cache with a bunch of values
> > +      IntStream.range(0,range).mapToDouble(value -> value).forEach(i -> cache.put(i, i + "-value"));
> > +      CacheSet<Map.Entry<Double, String>> entrySet = cache.entrySet();
> > +
> > +      int cnt = range;
> > +      for(int i = 0;i<cnt;i++){
>
> Formatting is off, please apply the formatter found at
> https://github.com/infinispan/infinispan/tree/master/ide-settings
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/infinispan/infinispan/pull/4288/files/48624b6203898692ae9652355ab079bc958dd1fa#r61585868>
>